ui/src/main/js/page/stocks/index.js

```diff
@@ -0,0 +1,180 @@
+'use strict';
+
+import React from 'react';
+import StockList from '../../component/stock-list/index.js';
+
+const ReactDOM = require('react-dom');
+const client = require('../../api/client');
+const follow = require('../../api/follow');
+const stompClient = require('../../websocket-listener');
+const when = require('when');
+const root = 'https://localhost/stock-service/api';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/modernmaster/forecaster/pullRequest?prid=7238741)
```suggestion
const root = "https://localhost/stock-service/api";
```